src/solver/optim-model-filler/include/antares/solver/optim-model-filler/ComponentFiller.h

```diff
@@ -79,7 +78,41 @@ class ComponentFiller: public Optimisation::LinearProblemApi::LinearProblemFille
     bool IsThisVariableTimeDependent(const std::string& var_id) const;
 
     const Study::SystemModel::Component& component_;
-    Expressions::Visitors::EvaluationContext evaluationContext_;
     const std::map<std::string, Study::SystemModel::Variable>& modelVariable_;
 };
+
+// TODO move me
+class VariablesBulkAddition
```

Comment:
as these are helper functions for ComponentFiller, I think you can create free static functions in ComponentFiller.cpp, or at least make this class private to ComponentFiller

Reply:
it was private, i decided to expose to unit test it